kubed-core/src/main/kotlin/kubed/interpolate/Interpolator.kt

```diff
@@ -1,8 +1,15 @@
 package kubed.interpolate
 
 import javafx.scene.paint.Color
-import kubed.color.ColorSpace
-import kubed.color.Rgb
+
+typealias Reinterpolator<T> = (Double) -> T
```

Comment:
After giving it some extra thought I settled on `Reinterpolator` instead of `Interpolator`. It makes for a nice symmetry in definitions below, and `Interpolator` class is already there in JavaFX.

Reply:
I prefer `Reinterpolator` as well, it's consistent with D3.